src/utils/fhirConstants.ts

```diff
@@ -4,44 +4,142 @@
 
 /**
  * Root URL for the MCode structure definition URLs.
+ * The Moonshot server uses STU1 of the MCode IG.
  */
 export const MCODE_STRUCTURE_DEFINITION = 'http://hl7.org/fhir/us/mcode/StructureDefinition/';
 
 /**
- * Primary cancer condition URL.
+ * Primary Cancer Condition URL.
  */
 export const MCODE_PRIMARY_CANCER_CONDITION = MCODE_STRUCTURE_DEFINITION + 'mcode-primary-cancer-condition';
 
 /**
- * The MCode resource URL for histology morphology.
+ * Secondary Cancer Condition URL.
+ */
+export const MCODE_SECONDARY_CANCER_CONDITION = MCODE_STRUCTURE_DEFINITION + 'mcode-secondary-cancer-condition';
+
+/**
+ * Histology Morphology Behavior Extension URL.
  */
 export const MCODE_HISTOLOGY_MORPHOLOGY_BEHAVIOR = MCODE_STRUCTURE_DEFINITION + 'mcode-histology-morphology-behavior';
 
 /**
- * The system URL for SNOMED codes.
+ * The system URLs for SNOMED, LOINC, and RxNorm codes.
  */
 export const SNOMED_CODE_URI = 'http://snomed.info/sct';
```

Comment:
Just to note, `/** */` are JavaDoc-style comments. These comments "document" only the next statement, so while documenting several constants in a row makes sense when looking at the code, when run through a program like [TypeDoc](https://typedoc.org/), it will only document the constant immediately afterwards.